docs-src/faq.md

```diff
@@ -1,6 +1,59 @@
 # Frequently Asked Questions (FAQ)
 
-*   Q: Will there be a default controller implementation (in this repo)?<br>
+*   **Q: Will Service API replace the Ingress API?<br>**
```

Comment:
I think this leads to some natural follow up questions:

* What is the difference between this API and the Ingress API?
* Why would I use this API instead of the Ingress API?
* Why would I use the Ingress API instead of this one?